index.html

```diff
@@ -184,91 +187,132 @@ <h1 class="card-header font-style on-small">
+						<img src="images/githublogo.png" alt="Github icon" />
+					</a>
+				</li>
+				<li>
+					<a href="#"><img src="images/Vector (2).svg" alt="two fingers"></a>
+				</li>
+			</ul>
+			<button type="button" class="resume-button button">Get my resume</button>
+			<div class="languages_for_web">
+				<section class="languages">
+					<a href="#" class="languages-heading">
+						<h3 class="font-style">Languages</h3>
+						<img class="vector" src="images/arrow-down.png" alt="arrow down" />
+					</a>
+					<ul class="lang">
+						<li>
+							<a class="language" href="#lang">
+								<img src="images/js.svg" alt="javascript logo" />
+								<h4 class="language-text javascript font-style">
+									JavaScript
+								</h4>
+							</a>
+						</li>
+						<li>
+							<a class="language m-left" href="#lang">
+								<img src="images/html.svg" alt="html logo" />
+								<h4 class="language-text html font-style">HTML</h4>
+							</a>
+						</li>
+						<li>
+							<a class="language m-left" href="#lang">
+								<img src="images/css.svg" alt="css logo" />
+								<h4 class="language-text html font-style">CSS</h4>
+							</a>
+						</li>
+					</ul>
+				</section>
+				<section class="frameworks">
+					<div class="framework-list">
+						<h3 class="font-style">Frameworks</h3>
+						<img class="vector" src="images/right-arrow.png" alt="arrow right" />
+					</div>
+					<div class="framework-list">
+						<h3 class="font-style">Skills</h3>
+						<img class="vector" src="images/right-arrow.png" alt="arrow right" />
+					</div>
+				</section>
+			</div>
 		</section>
-		<div class="languages_for_web">
-			<section class="languages">
-				<a href="#" class="languages-heading">
-					<h3 class="font-style">Languages</h3>
-					<img class="vector" src="images/arrow-down.png" alt="arrow down" />
-				</a>
-				<ul class="lang">
-					<li>
-						<a class="language" href="#lang">
-							<img src="images/js.svg" alt="javascript logo" />
-							<h4 class="language-text javascript font-style">
-								JavaScript
-							</h4>
-						</a>
-					</li>
-					<li>
-						<a class="language m-left" href="#lang">
-							<img src="images/html.svg" alt="html logo" />
-							<h4 class="language-text html font-style">HTML</h4>
-						</a>
-					</li>
-					<li>
-						<a class="language m-left" href="#lang">
-							<img src="images/css.svg" alt="css logo" />
-							<h4 class="language-text html font-style">CSS</h4>
-						</a>
-					</li>
-				</ul>
-			</section>
-			<section class="frameworks">
-				<div class="framework-list">
-					<h3 class="font-style">Frameworks</h3>
-					<img class="vector" src="images/right-arrow.png" alt="arrow right" />
-				</div>
-				<div class="framework-list">
-					<h3 class="font-style">Skills</h3>
-					<img class="vector" src="images/right-arrow.png" alt="arrow right" />
-				</div>
-			</section>
+
+	</div>
+
+	<footer id="contact" class="footer">

```

Comment:
- [**OPTIONAL** ] Semantically, I believe it is not acceptable to wrap your form within a `footer` tag because it is not a footer. Please use something like a 'div' instead.